src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} text={article.text} />
```

Comment:
Date from article object is not read. Try destructurisation.

Reply:
You mean use spread operator like this {...article}?